### ISSUE TITLE
Chart template error due to hard coded value

### ISSUE THREAD

**Issue by tonygilkerson** [NVIDIA/gpu-feature-discovery#39]
I found a hard coded value [here](https://github.com/NVIDIA/gpu-feature-discovery/blob/main/deployments/helm/gpu-feature-discovery/templates/daemonset.yml#L79) in the chart.  

I think you want this instead

```yaml
# Before (bug)
{{- if typeIs "bool" .Values.nfd.enableNodeFeatureApi }}
  - name: GFD_USE_NODE_FEATURE_API
    value: "true" #<--- Hard code
{{- end }}

# After
{{- if typeIs "bool" .Values.nfd.enableNodeFeatureApi }}
  - name: GFD_USE_NODE_FEATURE_API
    value: {{ .Values.nfd.enableNodeFeatureApi | quote }}
{{- end }}
```

**Comment by elezar** [comment]
cc @ArangoGutierrez

**Comment by ArangoGutierrez** [comment]
Will fix in tandem with the update of NFD charts to v0.13

**Comment by ArangoGutierrez** [comment]
https://gitlab.com/nvidia/kubernetes/gpu-feature-discovery/-/merge_requests/165

**Comment by elezar** [comment]
Hi @tonygilkerson. This should have been addressed in the v0.8.1 release. Could you confirm that this is resolved?

**Comment by tonygilkerson** [comment]
Yes, I just installed v0.8.1 and that did resolve the issue.  You can close this issue.  Thanks
